internal/pkg/db/db.go

```diff
@@ -238,6 +270,30 @@ func (backend *Backend) FilterDocumentsByTag(documents []*sbom.Document, tags ..
 	return documents, nil
 }
 
+func (backend *Backend) MoveAlias(revision, base *sbom.Document) error {
```

Comment:
Something like the name `UpdateAlias` might make more sense, how would you feel about that?

Reply:
Yeah I also thought about TransferAlias, since thats the goal of the function

Reply:
I think I want to revisit this. To me, `TransferAlias` doesn't seem to really describe its purpose. When I think "transfer", I think copy data from one place to another. This is more like updating the underlying reference for a pointer. So in my opinion, `UpdateAliasReference` makes the most sense, maybe abbreviated to just `UpdateAliasRef` or (less preferable) `UpdateAlias`